src/Frontend/Core/Engine/Header.php

```diff
@@ -132,7 +132,11 @@ public function addCSS($file, $minify = true, $addTimestamp = null)
         $file = (string) $file;
         $minify = (bool) $minify;
         $addTimestamp = (bool) $addTimestamp;
-        $file = Theme::getPath($file);
+
+        // get file path
+        if (mb_substr($file, 0, 4) != 'http') {
```

Comment:
But I don't consider this to be a blocking issue if @jessedobbelaere wouldn't have time to change it now :)